src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1403,4 +1403,13 @@ This version used in: {2}</value>
   <data name="Formatting_document" xml:space="preserve">
     <value>Formatting document</value>
   </data>
+  <data name="Formatting_analyzer_code_fix" xml:space="preserve">
+    <value>Fix formatting</value>
+  </data>
+  <data name="Formatting_analyzer_message" xml:space="preserve">
+    <value>Fix formatting</value>
+  </data>
+  <data name="Formatting_analyzer_title" xml:space="preserve">
+    <value>Fix formatting</value>
```

Comment:
this doesn't seem to match a pattern we've ever used elsewhere (including times when our message and title are the same)?  Is this necessary?  Has there been a customer issue or loc-request for us to do this?

Reply:
I didn't want to keep renaming them. After we settle on the text for each of the three cases, I'll rename the keys to match the existing pattern. 😄

Reply:
Ah ok :) 👍

Reply:
Fixed in 031cd695031feab5d388bc8a15e297fb19a584d0